pinot-plugins/pinot-file-system/pinot-hdfs/src/main/java/org/apache/pinot/plugin/filesystem/HadoopPinotFS.java

```diff
@@ -60,6 +62,7 @@ public void init(Configuration config) {
       _hadoopConf = getConf(config.getString(HADOOP_CONF_PATH));
       authenticate(_hadoopConf, config);
       _hadoopFS = org.apache.hadoop.fs.FileSystem.get(_hadoopConf);
+      _hadoopFS.setWriteChecksum((config.getBoolean(WRITE_CHECKSUM, false)));
```

Comment:
Why do we need to set it to false?

Reply:
This will still do checksum but not write the `.crc` file to directory.